.taskcluster.yml

```diff
@@ -5,385 +5,278 @@
-                          description: 'The task that creates all of the other tasks in the task graph'
-                  provisionerId: "app-services-${level}"
-                  workerType: "decision"
-                  tags:
-                    $if: 'tasks_for in ["github-push", "github-pull-request"]'
-                    then:
-                      kind: decision-task
-                  routes:
-                    $flattenDeep:
-                      - checks
-                      - $if: 'level == "3"'
-                        then:
-                          # TODO Bug 1601928: Make this scope fork-friendly once ${project} is better defined. This will enable
-                          # staging release promotion on forks.
-                          - $if: 'tasks_for == "github-push"'
-                            then:
-                              - index.project.${project}.v2.branch.${short_head_branch}.latest.taskgraph.decision
-                              - index.project.${project}.v2.branch.${short_head_branch}.revision.${head_sha}.taskgraph.decision
-                          - $if: 'tasks_for == "cron"'
-                            then:
-                              # cron context provides ${head_branch} as a short one
-                              - index.project.${project}.v2.branch.${head_branch}.latest.taskgraph.decision-${cron.job_name}
-                              - index.project.${project}.v2.branch.${head_branch}.revision.${head_sha}.taskgraph.decision-${cron.job_name}
-                              - index.project.${project}.v2.branch.${head_branch}.revision.${head_sha}.taskgraph.cron.${ownTaskId}
-                  scopes:
-                    $if: 'tasks_for == "github-push"'
-                    then:
-                      # `https://` is 8 characters so, ${repoUrl[8:]} is the repository without the protocol.
-                      - 'assume:repo:${repoUrl[8:]}:branch:${short_head_branch}'
-                    else:
-                      $if: 'tasks_for == "github-pull-request"'
-                      then:
-                        - 'assume:repo:github.com/${event.pull_request.base.repo.full_name}:pull-request'
-                      else:
-                        $if: 'tasks_for == "github-release"'
-                        then:
-                          - 'assume:repo:${repoUrl[8:]}:release'
-                        else:
-                          $if: 'tasks_for == "action"'
+                      $if: 'tasks_for == "cron"'
+                      then: '${ownTaskId}'
+              pullRequestAction:
+                  $if: 'tasks_for == "github-pull-request"'
+                  then: ${event.action}
+                  else: 'UNDEFINED'
+              releaseAction:
+                  $if: 'tasks_for == "github-release"'
+                  then: ${event.action}
+                  else: 'UNDEFINED'
+          in:
+              $if: >
+                tasks_for in ["action", "cron"]
```

Comment:
Ditched the formerly `matched` commands to this, to match what we have in the other repos.